relay-ourlogs/src/otel_to_sentry.rs

```diff
@@ -0,0 +1,340 @@
+//! Transforms OpenTelemetry logs to Sentry logs.
+//!
+//! This module provides functionality to convert OpenTelemetry log records
+//! into Sentry's internal log format (`OurLog`).
+
+use chrono::{TimeZone, Utc};
+use opentelemetry_proto::tonic::common::v1::any_value::Value as OtelValue;
+use opentelemetry_proto::tonic::logs::v1::LogRecord as OtelLogRecord;
+
+use relay_event_schema::protocol::{Attributes, OurLog, OurLogLevel, SpanId, Timestamp, TraceId};
+use relay_otel::otel_value_to_attribute;
+use relay_protocol::{Annotated, Error, Object};
+
+fn otel_value_to_string(value: Option<opentelemetry_proto::tonic::common::v1::AnyValue>) -> String {
```

Comment:
Should this return `Option<String>`? So `body` can later be an empty `Annotated`.